packages/protobuf/scripts/bootstrap-inject.mjs

```diff
@@ -158,7 +158,12 @@ async function processFile(filePath, content, descriptorProto, upstream) {
         for (const def of featureSetDefaults.defaults) {
           lines.push(`  // ${Edition[def.edition]}`);
           lines.push(`  ${def.edition}: {`);
-          const r = reflect(FeatureSetDesc, def.features);
+          const r = reflect(
+            FeatureSetDesc,
+            featureSetHasAllSet(def.overridableFeatures)
+              ? def.overridableFeatures
+              : def.fixedFeatures,
+          );
```

Comment:
The message google.protobuf.FeatureSetDefaults.FeatureSetEditionDefault has changed in v27.0-rc1: The field `features` is replaced by the fields `overridable_features` and `fixed_features`, so we have to update our script
 that embeds edition defaults.

This was caught by our assertions, so the script is doing its job.